### DISCUSSION TITLE
Mobility API and tourism API both expose Weather Realtime data

### DISCUSSION THREAD

**Discussion by BlueRyse** [noi-techpark/it.bz.opendatahub.databrowser#179] [General]
As briefly discussed in the Sprint meeting, both of the APIs expose the same data:

In mobility:
https://mobility.api.opendatahub.com/v2/flat,node/MeteoStation/*/latest

In Tourism:
https://tourism.api.opendatahub.bz.it/v1/Weather/Realtime?language=en

We decided to start a discussion to see how we want to proceed, and what would the outcomes be of this new finding.

**Comment by sseppi** [comment]
@RudiThoeni do you have statistics about the usage of the API call in the Tourism domain?

**Reply by RudiThoeni** [reply]
i think the realtime route is not used but i will check all logs
